### ISSUE TITLE
Per world teleport permissions

### ISSUE THREAD

**Issue by Gwittmann** [Multiverse/Multiverse-Core#2256]
I want to allow teleporting to the nether by using a nether portal as you would in vanilla, but disallow teleporting to the nether via command. I've tried setting the multiverse.access.world_nether permission to false, but it completely blocks the player from entering the nether via portal or teleport command, when I just want to disable the latter. I've also tried setting multiverse.teleport.self.w to false, and while this works for the nether, it disables teleports for all worlds, and I want users to be able to use the teleport command for creative.

In summary of what I'm trying to do:
While in Survival:
Teleport command to nether: false
Portal to nether: true
Teleport command to creative: true

While in nether:
Teleport command to survival: false
Portal to survival: true
Teleport command to creative: false

While in creative:
Teleport command to survival: true
Teleport command to nether: false

**Comment by benwoo1110** [comment]
Sadly, we have not implemented per world teleport perms, hope it will be implement soon:tm:

**Comment by benwoo1110** [comment]
Combine with #1292
